PGM/src/main/java/tc/oc/pgm/mutation/types/targetable/BomberMutation.java

```diff
@@ -36,6 +36,7 @@ public void target(List<MatchPlayer> players) {
                 TNTPrimed tnt = spawn(location, TNTPrimed.class);
                 tnt.setGlowing(true);
                 tnt.setIsIncendiary(false);
+                tnt.setYield(0);
```

Comment:
setYield changes the tnt's explosion radius. does the tnt still do damage to players?

Reply:
Yes, TNT still does damage to players. I thought yield stopped how many blocks dropped? When I tested the yield and mutation bomber through XML, it still exploded the landscape but prevented blocks from dropping.

Reply:
Yield changes percent of entity drops from a scalar [0,1].